subgraphs/across-v2/src/handlers/liquidity.ts

```diff
@@ -68,7 +70,12 @@ export function handleLiquidityAdded(event: LiquidityAdded): void {
   }
 
   // output token price
-  pool.pool.outputTokenPriceUSD = token.lastPriceUSD;
+  const exchangeRate = bigIntToBigDecimal(event.params.amount).div(
+    bigIntToBigDecimal(event.params.lpTokensMinted)
+  );
+  pool.pool.outputTokenPriceUSD = tokenPricer
+    .getTokenPrice(token)
```

Comment:
Had to do use tokenPricer instead of `token.lastPriceUSD` because it wasn't available (null wasm errors). Looking thru the SDK, seems like token price should be set and saved when adding volume or refreshing TVL but I don't see that happening here.